src/scss/main.scss

```diff
@@ -107,6 +107,42 @@ main.container {
   width: auto; margin: 0 2%;
   max-width: 60em;
 
+  .container {
+    text-align: center;
+    width: 80%;
+  }
+  a {
+    text-decoration: none;
+    color: #0192b5;
+  }
+
+  a:hover {
+    text-decoration: underline;
+  }
+  .textarea {
+    outline: 1px solid black;
+    margin-left: 20vw;
+    margin-top: -16px;
+    z-index: -1;
+    position: relative;
+  }
```

Comment:
Try to avoid using classnames that are also the names of HTML elements. Also better to have a classname that reflects the semantic meaning of the element on the page than a generic name. Avoid using `position` unless you really need it (why do you need it here?) and don't overuse `vw` units. Rarely do I want a typographic margin to be relative to the viewport.